jerry-core/jerry.cpp

```diff
@@ -899,6 +900,57 @@ bool jerry_api_get_object_field_value (jerry_api_object_t *object_p,
 }
 
 /**
+ * Applies the given function to the every fields in the objects
+ *
+ */
+bool
+jerry_api_foreach_object_field (jerry_api_object_t *object_p, /**< object */
+                               jerry_object_field_foreach_t foreach_p, /**< foreach function */
+                               void *user_data) /**< user data for foreach function */
```

Comment:
As this is a variable of pointer type, its better to add `_p` suffix for it - `user_data_p`.